lib/jxl/dec_modular.cc

```diff
@@ -288,6 +288,7 @@ Status ModularFrameDecoder::DecodeGroup(const Rect& rect, BitReader* reader,
     }
   }
   if (zerofill && use_full_image) return true;
+  if (gi.channel.empty()) return true;
```

Comment:
please add a comment explaining why is this check here. The idea is to provide more context to the next person who needs to debug an issue in this code.

Reply:
Added a comment